src/software/sensor_fusion/sensor_fusion_test.cpp

```diff
@@ -294,6 +341,93 @@ class SensorFusionTest : public ::testing::Test
     }
 };
 
+TEST_F(SensorFusionTest, test_making_robot_move_capability_unavailable_from_error_code)
+{
+    SensorProto sensor_msg;
+    auto ssl_wrapper_packet =
+        createSSLWrapperPacket(std::move(geom_data), initDetectionFrame());
+    *(sensor_msg.mutable_ssl_vision_msg()) = *ssl_wrapper_packet;
+    *(sensor_msg.add_robot_status_msgs())  = *robot_status_msg_wheel_motor_hot;
+    sensor_fusion.processSensorProto(sensor_msg);
+
+    std::optional<Robot> robot =
```

Comment:
I noticed that for this test case you get the robot directly, but in your other tests you get the team first, and then get the robot from the team. Maybe you switched your method by accident? Just checking in case you didn't mean to do this :)

Reply:
I will update them to match. Thanks!